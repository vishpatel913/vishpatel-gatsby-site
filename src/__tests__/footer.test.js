// import React from "react";
// import renderWithTheme from "../utils/test-renderer";
// import Footer from "../components/footer";

describe("Footer", () => {
  it("renders correctly, well meh", () => {
    // const tree = renderWithTheme(<Footer />).toJSON();
    // expect(tree).toMatchSnapshot();
  });
});
