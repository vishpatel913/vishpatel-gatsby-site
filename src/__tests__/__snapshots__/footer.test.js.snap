// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
.c0 {
  margin: 0 auto;
  text-align: center;
  max-width: 1200px;
  padding: 0.5rem 2rem 1rem;
}

.c1 {
  max-width: 100px;
  width: 100%;
  -webkit-filter: opacity(0.7);
  filter: opacity(0.7);
  -webkit-transition: all 0.2s ease 0s;
  transition: all 0.2s ease 0s;
}

.c1:hover {
  -webkit-filter: opacity(1);
  filter: opacity(1);
}

@media (max-width:768px) {
  .c0 {
    padding: 1rem;
  }
}

<footer
  className="c0"
>
  <a
    href="https://www.contentful.com/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="Powered by Contentful"
      className="c1"
      src="https://images.ctfassets.net/fo9twyrwpveg/44baP9Gtm8qE2Umm8CQwQk/c43325463d1cb5db2ef97fca0788ea55/PoweredByContentful_LightBackground.svg"
    />
  </a>
</footer>
`;
